examples/enclosures.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 ###
-# (C) Copyright (2012-2017) Hewlett Packard Enterprise Development LP
+# (C) Copyright (2012-2019) Hewlett Packard Enterprise Development LP
```

Comment:
We have to update the [License](https://github.com/HewlettPackard/python-hpOneView/blob/master/LICENSE) file first.

Reply:
Done.